Assets/Scripts/Game/CombatManager.cs

```diff
@@ -51,6 +68,21 @@ public void InitialiseCombat(CarriageData _currentCarriage)
             //ASSIGN TO ACTIVE ENEMIES
             activeEnemies.Add(_spawnedEnemy);
         }
+        #endregion
+
+        #region SpawnBoss
+        if (currentCarriage._isSpecialCarriage == CarriageData.SpecialCarriageExceptions.Vault && !isBossAlive)
+        {
+            //CHOOSE SPAWNER
+            Vector3 _bossSpawnPosition = currentCarriage.transform.Find("BossSpawner").transform.position;
+
+            //SPAWN ENEMY
+            GameObject _boss = Resources.Load<GameObject>("Boss");
+            Instantiate(_boss, _bossSpawnPosition, Quaternion.identity);
+        }
+
+
```

Comment:
too many blank lines

Reply:
It has a #region line in the actual unity code, for organisation. It's not appearing here on this end for some reason?